src/providers/azure.rs

```diff
@@ -313,6 +313,7 @@ impl super::Launcher for RegionLauncher {
                     nickname: name.clone(),
                     public_dns: ip.clone(),
                     public_ip: ip.clone(),
+                    private_ip: None,
```

Comment:
I'm surprised that Azure VMs do not also have internal IPs?

Reply:
It does - I have to figure out how to query them via the cli.